server/lib/src/idm/oauth2.rs

```diff
@@ -238,6 +239,8 @@ pub struct Oauth2RS {
     scopes_supported: BTreeSet<String>,
     prefer_short_username: bool,
     type_: OauthRSType,
+    /// the display image for the OAuth2 Resource
+    display_image: Option<ImageValue>,
```

Comment:
Does this need to be here in memory? I think we are serving the image from the entry directly, rather than the in-memory oauth2 resource here, so I think this should just be a bool for if the image is set or not?

Reply:
Done!